core/common/src/main/java/alluxio/collections/NonUniqueFieldIndex.java

```diff
@@ -68,6 +69,7 @@ public boolean add(T object) {
   }
 
   @Override
+  @SuppressWarnings("nullness")
```

Comment:
If the warning suppression is removed, the Checker Framework report the following error:
```
found   : T extends @Initialized @Nullable Object
required: @Initialized @NonNull Object
```
That error seems to be a false positive since it's clear from the method signature that the variable ```object``` it's supposed to be ```non-null```. I replaced the suppression with an assertion to make it more clear.